cvat/apps/engine/models.py

```diff
@@ -430,6 +432,26 @@ class Meta:
     def touch(self) -> None:
         self.save(update_fields=["updated_date"])
 
+class ABCModelMeta(ABCMeta, ModelBase):
+    pass
+
+class _FileSystemRelatedModel(models.Model, metaclass=ABCModelMeta):
```

Comment:
Why this class has an underscore prefix? Models inherit it, and it's functions are supposed to be used by model clients.

Reply:
> Models inherit it, and it's functions are supposed to be used by model clients.

That's true, but I named this class with an underscore to indicate that it will not be imported into other modules.

Reply:
Do you mean will not be imported if someone uses `from models import *`? It's already discouraged in the project by pylint.